dpe/src/context.rs

```diff
@@ -113,16 +113,21 @@ pub struct ContextHandle(pub [u8; ContextHandle::SIZE]);
 
 impl ContextHandle {
     pub const SIZE: usize = 16;
-    const DEFAULT: [u8; Self::SIZE] = [0; Self::SIZE];
+    const DEFAULT: ContextHandle = ContextHandle([0; Self::SIZE]);
 
     /// Returns the default context handle.
     pub const fn default() -> ContextHandle {
-        ContextHandle(Self::DEFAULT)
+        Self::DEFAULT
     }
 
     /// Whether the handle is the default context handle.
     pub fn is_default(&self) -> bool {
-        constant_time_eq(&self.0, &Self::DEFAULT)
+        self.equals(&Self::DEFAULT)
+    }
+
+    #[inline(never)]
+    pub fn equals(&self, other: &ContextHandle) -> bool {
```

Comment:
optional: I'm mildly interested to know if implementing the [Eq](https://doc.rust-lang.org/std/cmp/trait.Eq.html) trait does something to the firmware size. This would have the added benefit of being able to use the `==` operator as well.

Reply:
Will look into this as a follow up. Thanks!